pkg/cmd/cli/cmd/newapp.go

```diff
@@ -142,6 +142,7 @@ func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer)
 		},
 	}
 
+	cmd.Flags().BoolVar(&config.AsTestDeployment, "as-test", config.AsTestDeployment, "If true create this application as a test deployment, which validates that the deployment succeeds and then scales down.")
```

Comment:
so you're proposing you cannot set it as part of invoking new-app itself?  Given the primary use case for creating a test deployment is so it can tag an image when it passes, it seems like i should be able to do that in a single command.